templates/solutions/ai/index.html

```diff
@@ -1,29 +1,32 @@
 {% extends 'base_index.html' %}
 
+{% from "macros/quote_wrapper.jinja" import quote_wrapper %}
+
 {% block meta_copydoc %}
   https://docs.google.com/document/d/1AtAApqxX9cbWf4a3cakEYbk7OoTfdBBi7FlA3WZz5ZM/edit
-{% endblock meta_copydoc %}
+{% endblock %}
 
 {% block title %}Open source AI for the enterprise{% endblock %}
 
 {% block meta_description %}
   Innovate quickly, securely, and cost-effectively with open source AI for the enterprise &mdash; all on a single stack.
 {% endblock %}
 
-{% block body_class %}
-  is-paper
-{% endblock body_class %}
+{% block body_class %}is-paper{% endblock %}
 
 {% block content %}
-
   <section class="p-section--hero">
-    <div class="row--50-50 p-section">
+    <div class="row--50-50">
       <div class="col">
-        <h1 class="p-section--shallow">Simplify your enterprise AI journey with trusted open source</h1>
+        <h1 class="p-section--shallow">
+          Enterprise AI solutions
+          <br />
+          built with trusted open source
+        </h1>
         <p>
-          Innovate at speed with secure and compliant open source. Get all the AI tooling you need in an integrated stack at a predictable cost.
+          Innovate at speed with compliant open source – from workstations to clouds and smart devices. Get all the enterprise AI solutions you need from a trusted partner at a predictable cost.
         </p>
-        <hr class="p-rule" />
+        <hr class="p-rule--muted" />
         <a class="p-button--positive"
            href="/contact-us"
            aria-controls="ai-modal">Contact Canonical</a>
```

Comment:
I think we can repalce this section with hero_macro https://vanillaframework.io/docs/patterns/hero